tests/functions/New-TssSession.Tests.ps1

```diff
@@ -98,5 +98,8 @@ Describe "$commandName works" {
             $currentTime = [datetime]::UtcNow
             $sessionAccessToken.StartTime | Should -BeLessOrEqual $currentTime
         }
+        It "Sets correct TokenType" {
```

Comment:
Please add this for the Credential context as well for consistency.

Reply:
in https://github.com/thycotic-ps/thycotic.secretserver/pull/73/commits/282723876f9f3ca37e8e759f82c408b676e0567e